ui/apps/platform/src/Containers/AccessControl/AuthProviders/authProviders.utils.ts

```diff
@@ -191,7 +191,6 @@ export function mergeGroupsWithAuthProviders(
             }
         });
     }
-
```

Comment:
nit: Try not to let your editor take out single blank lines. This one give some visual separation of the return statement from the logic of the function, which aid in "scanning" the code late.